sql/2021/mobile-web/credential_management_api_usage.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+# Usage of Credential Manager
+SELECT DISTINCT
+  client,
+  feature,
+  num_urls AS freq,
+  total_urls AS total,
+  pct_urls AS pct
+FROM
+  `httparchive.blink_features.usage`
+WHERE
+  yyyymmdd = '20210701' AND
+  feature = 'CredentialManagerGet'
```

Comment:
Is it good practice to `OR` this with the blink id? Think some of the older years didn't have the name and/or do these change? But probably not a big deal as been in there for a while and only looking at a single month here anyway.

Reply:
What is the blink id? I've only ever seen it blank on sample rows in the table

Reply:
Actually looks like I had it the wrong way round - the id is sometimes blank for earlier years.

I guess using the id might be better in case the name is every changed slightly in future years? With an OR for the existing name for years in the past?

Anyway doesn't matter too much in this case.

Reply:
Resolving this one as fine as is.